samTemplates/amazon-connect-interactive-messages-example-lev-v1/template.yaml

```diff
@@ -0,0 +1,35 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+
+Metadata:
+  AWS::ServerlessRepo::Application:
+    Name: amazon-connect-interactive-messages-example
+    Description: Creates AWS Lambda function to be used as an initialization and validation code hook in Amazon Lex Bot for enabling interactive messages in Amazon Connect Chat.
+    Author: Amazon Connect CX Innovations
+    SpdxLicenseId: MIT-0
+    LicenseUrl: LICENSE.txt
+    ReadmeUrl: README.md
+    Labels: ['connect', 'chat', 'interactive-messaging', 'contact-center', 'lambda', 'lex', 'bot']
+    HomePageUrl: https://github.com/amazon-connect/amazon-connect-chat-ui-examples/tree/master/samTemplates/amazon-connect-interactive-messages-example
+    SemanticVersion: 0.0.1
+    SourceCodeUrl: https://github.com/amazon-connect/amazon-connect-chat-ui-examples/tree/master/samTemplates/amazon-connect-interactive-messages-example
+  
+Globals:
+  Function:
+    Timeout: 30
+
+Resources:
+  InteractiveMessagingLambdaFunction:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Deprecated runtime (nodejs12.x) specified. Updating disabled since 2022-12-14. Please consider updating to nodejs16.x

Reply:
thank you for leaving this comment. Will upgrade in https://github.com/amazon-connect/amazon-connect-chat-ui-examples/pull/186